packages/server/lib/browsers/index.js

```diff
@@ -149,6 +149,8 @@ module.exports = {
         onBrowserClose () {},
       })
 
+      ctx.browser.setBrowserStatus('opening')
```

Comment:
Should we add an assertion in the test to see if this was set (easy if you use the testDataContext, I think - see my other comment).

Reply:
Sure I'll give it a go

Reply:
Done in [3256e0f](https://github.com/cypress-io/cypress/pull/20563/commits/3256e0f3565f2c04d01114bd512563748fb6c953)